contracts/abi/LineaRollupV6.0.abi

```diff
@@ -557,40 +557,40 @@
       {
         "indexed": true,
         "internalType": "uint256",
-        "name": "lastBlockFinalized",
+        "name": "startBlockNumber",
+        "type": "uint256"
+      },
+      {
+        "indexed": true,
+        "internalType": "uint256",
+        "name": "endBlockNumber",
         "type": "uint256"
       },
       {
         "indexed": true,
         "internalType": "bytes32",
-        "name": "startingRootHash",
+        "name": "shnarf",
         "type": "bytes32"
       },
       {
-        "indexed": true,
+        "indexed": false,
         "internalType": "bytes32",
-        "name": "finalRootHash",
+        "name": "parentStateRootHash",
         "type": "bytes32"
       },
       {
         "indexed": false,
-        "internalType": "bool",
-        "name": "withProof",
-        "type": "bool"
+        "internalType": "bytes32",
+        "name": "finalRootHash",
```

Comment:
this shall be final**State**RootHash,  to 1) be consistent with parentStateRootHash, 2) less ambiguous.

Reply:
This has been done as well - new ABI will be pushed shortly